lib/exchanges.ts

```diff
@@ -0,0 +1,24 @@
+export enum Exchange {
```

Comment:
does this enum buy us anything vs having a union type like `'blur' | 'opensea' | ...`? Seems like we just use it to key into this object and it makes it more verbose.